android/keyple-plugin/android-nfc/src/main/kotlin/org/eclipse/keyple/plugin/android/nfc/AbstractAndroidNfcReader.kt

```diff
@@ -135,15 +136,20 @@ internal object AndroidNfcReaderImpl : AbstractObservableLocalReader(AndroidNfcR
     @Throws(IllegalArgumentException::class)
     override fun setParameter(key: String, value: String) {
```

Comment:
The setParameter command is no more "overridden", because the corresponding method has been removed from the interface org.eclipse.keyple.core.service.Reader.

A better way for the NFC plugin could be to declare in the interface AndroidNfcReader: some public methods to manage settings specific to the NFC plugin.